README.md

```diff
@@ -15,10 +15,23 @@ Tanzu Toolkit for Visual Studio is an extension for Visual Studio (2022 & 2019)
 - [Remotely debug](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/wiki/Remote-Debugging-TAS-Apps) apps running on TAS within Visual Studio
 
 ## Install
-- To install this extension, you will need Visual Studio 2022 or 2019.
-  - The extension can then be found on both versions' Visual Studio Marketplace, by searching for Tanzu Net Experience.
-- Download the `Tanzu.Toolkit.VisualStudio.vsix` file & double-click to prompt an installation into your Visual Studio instance.
-- To uninstall this extension from Visual Studio, visit the Visual Studio Extension Manager found under the `Extensions` menu.
+To install this extension, you will need:
+  1. Visual Studio version 2022 or 2019
+  2. The `Tanzu.Toolkit.VisualStudio.vsix` extension file, downloadable from any of these sources:
+      - The Visual Studio Marketplace:
+        - through the IDE by searching "Tanzu" in the "Manage Extensions" window
+        - through the website: [VS 2022](https://marketplace.visualstudio.com/items?itemName=TanzuNETExperience.TanzuToolkitForVisualStudio2022), [VS 2019](https://marketplace.visualstudio.com/items?itemName=TanzuNETExperience.TanzuToolkitForVisualStudio2019)
+      - vsixgallery.com: [VS 2022](https://www.vsixgallery.com/extension/TanzuToolkitForVisualStudio2022.ff7b6f3e-0410-4ff9-a40a-a719ee9da901), [VS 2019](https://www.vsixgallery.com/extension/TanzuToolkitForVisualStudio2019.ff7b6f3e-0410-4ff9-a40a-a719ee9da901)
```

Comment:
We should separate this out to a separate section or indicate that these are the "latest" builds and probably won't match the VS Marketplace or releases section